docs/DeveloperGuide.md

```diff
@@ -305,6 +305,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `*`      | Book recommender who used the application for a long time | recommend books using stored statistics of most frequently requested books    | I can make better recommendations for students that come to us                                                                        |
 | `*`      | Impatient user                                            | be able to experience reasonable response time while using the application    | I can do my job more efficiently                                                                                                      |
 | `*`      | Easily overwhelmed request handler                        | be able to prioritize my book requests                                        | I can effectively handle requests                                                                                                     |
+| `***`    | Librarian                                                 | be able to view patron and book details through a GUI                         |                                                                                                                                       |
+| `***`    | Librarian                                                 | have visual feedback of the commands I executed                               | I will know if my commands have been successfully executed                                                                            |
```

Comment:
Maybe can move these to the top cause it has been assigned a higher priority.

Reply:
Point taken, I think our user stories will need to rerank them after everyone added a priority for their assigned user stories. Then, we can create a branch to update UG.